File: net/giftoxic/files/patch-src-utils.c
--- src/utils.c.orig	Tue May 27 21:37:18 2003
+++ src/utils.c	Sun Dec 28 14:25:22 2003
@@ -366,6 +366,8 @@
 		
 	gulong divisor = 1;
 	gint type = 0;
+	gulong mod;
+	gulong nice;
 		
 	if (size >= 1000000){
 		divisor = 1000000;
@@ -375,8 +377,8 @@
 		type = 1;
 	}
 	
-	gulong mod = size % divisor;
-	gulong nice = size / divisor;
+	mod = size % divisor;
+	nice = size / divisor;
 	strcpy(format, (mod == 0)?"%.0f":"%.2f");
 	strcat(format, (nice == 1 && mod == 0?gettext(qtys[type][0]):gettext(qtys[type][1])));
 	return g_strdup_printf(format, (gfloat) size / divisor);

File: net-im/jabber-pymsn/files/patch-src_utils.py
--- src/utils.py.orig	Mon Dec  5 16:41:53 2005
+++ src/utils.py	Mon Dec  5 16:42:18 2005
@@ -180,4 +180,8 @@
 	def flush(self):
 		self.lst = []
 
-
+def doPath(path):
+	if(path and path[0] == "/"):
+		return path
+	else:
+		return "../" + path
